.gitignore

```diff
@@ -3,6 +3,7 @@ tmp
 *~
 \.\#*
 \#*#
+*.depends
```

Comment:
what is .depends?

Reply:
An extension usually used for files generated by `ocamldep` and similar tools